TASK1/image.php

```diff
@@ -0,0 +1,26 @@
+ <?php
+
+  // Check if submit is set in post method then display hello fullname.
+  if (isset($_POST['submit'])) {
+    // Check if submit is set.
+    $upload = FALSE;
+    if (isset($_FILES)) {
+      // If files are uploaded on submit then.
+      $fileName = './uploaded/' . $_FILES['image']['name'];
```

Comment:
For PHP, only class object variables should be camelcased.
Rest variables defined in simple php files or as function arguments or inside functions should all be snake cased.